charts/nri-bundle/Chart.yaml

```diff
@@ -45,13 +45,21 @@ dependencies:
     version: 1.0.0
 
   - name: kube-state-metrics
-    version: 4.23.0
+    version: 4.25.0
     condition: ksm.enabled,kube-state-metrics.enabled
     repository: https://prometheus-community.github.io/helm-charts
+    alias: kube-state-metrics
+
+  - name: kube-state-metrics
+    version: 4.23.0
+    condition: ksm_v26.enabled,kube-state-metrics-v26.enabled
+    repository: https://prometheus-community.github.io/helm-charts
+    alias: kube-state-metrics-v26
```

Comment:
It seems like we use a condition named ksm_v26 to enable KSM 4.23. Is this correct? Moreover the other KSM is not 4.26 either, but 4.25.
I would recommend to use the full chart name and number, e.g. `kube-state-metrics-v4-23` for both the alias and the dependency.

Reply:
I disagree with both of the approaches as the user should then know which version of KSM is needed for their cluster. 

I would propose to alias it to something like `kube-state-metric-for-kubernetes-pre-1-26`. Maybe `kube-state-metric-pre-1-26` but I prefer the long one because it is self-explanatory.

Reply:
I like the `kube-state-metrics-for-k8s-pre-1-23` naming suggestion.